modules/data.atmosphere/vignettes/tdm_downscaling.Rmd

```diff
@@ -0,0 +1,55 @@
+Temporally Downscale Meteorology
+===============
+
+
+### Subdaily Training Data
+
+Examples:
+
+* Fluxnet2015 30-minute/hourly
+* Ameriflux 30-minute/hourly
+
+### Extract Training Data and Merge All Years Into 1 File 
+
+```{r}
+library(PEcAn.data.atmosphere)
+library(PEcAn.DB)
+download.Fluxnet2015(sitename = 'Niwot Ridge Forest/LTER NWT1 (US-NR1)', username = 'pecan', overwrite = FALSE, outfolder = '~/Example', 
+   start_date = '2000/01/01', end_date = '2014/12/31')
```

Comment:
start_date, end_date, lat, lon, in,prefix, outfolder's root folder, etc. are all things that should be variables, not hard coded. This is less buggy and will make it far easier for others to apply this example at new sites. I'd recommend defining all these in their own r code block that'd be under the previous header.